war/src/main/java/io/lumeer/engine/controller/OrganizationFacade.java

```diff
@@ -89,6 +90,18 @@ public String readOrganizationName(final String organizationId) {
    }
 
    /**
+    * Reads the organization id according to its name.
+    *
+    * @param organizationName
+    *       name of the organization
+    * @return id of the given organization
+    */
+   public String readOrganizationId(final String organizationName) {
```

Comment:
Task description https://lumeer.myjetbrains.com/youtrack/issue/LMR-135 says "read organization id according to organization name" so I thought it has to be unique.